functions/register/main.py

```diff
@@ -36,7 +36,7 @@ def register(request):
 
     request_data = request.get_json()
     msisdn = request_data["msisdn"]
-    ip = request.headers.get('X-Forwarded-For')
+    ip = request.headers.get('X-Forwarded-For').split(',')[-1]
```

Comment:
Jeśli [dokumentacja](https://cloud.google.com/appengine/docs/flexible/python/reference/request-headers) mówi `the first IP in this list is generally the IP of the client that created the request. ` to chyba chcemy `ip = request.headers.get('X-Forwarded-For').split(',')[0]`?
Ja bym to jeszcze `strip()` na koniec

Reply:
Zgodnie z https://github.com/ProteGO-app/backend/issues/49 branie ostatniego IP jest OK.